src/components/ConfigProvider/ConfigProvider.stories.tsx

```diff
@@ -201,6 +202,32 @@ const ThemedComponents: FC = () => {
                     />
                 </div>
             </Stack>
+            <div>
+                <TextArea
+                    enableExpand
+                    labelProps={{ text: 'Var Theme (updates onBlur)' }}
```

Comment:
An apply theme button would be more straight forward as opposed to manually adding textarea content onBlur? Another idea would be to a few forms in modals triggered by buttons - like [Custom button theme] [Custom table theme]

The Modal would contain a list inputs with the current css var values, then the person may pick and choose which to manually override at a glance as opposed to figuring out what the css vars are via the web inspector. The form would then send that back to the text area and apply the theme.

Something to consider.

Reply:
per office hours discussion, implemented the dialog to eliminate hacky onBlur implementation and have linked to the scss file for reference as to whats available